autoextract_spiders/spiders/autoextract_spider.py

```diff
@@ -169,11 +169,11 @@ def make_extract_request(self, url, meta=None, check_page_type=True):
                                  errback=self.errback_item,
                                  dont_filter=True)
 
-        if check_page_type and self.page_type == 'article':
+        if check_page_type and self.page_type in ('article', 'jobPosting'):
             if maybe_is_article(url):
```

Comment:
I think it could be better to have a separate function for checking if a page can be a job page or not. For example, about and contact may contain job postings, but they're filtered out here.

Reply:
Makes sense. Created a new function for jobs. I re-organized that part of the code to make it more robust in the future.
Also added a test.